index.bs

```diff
@@ -194,8 +194,8 @@ Windows can be placed on specific screens using the coordinates provided in the
 function openAndCenterWindow(url, screenDetailed, width, height) {
 
   // Compute coordinates relative to the target screen's coordinates.
-  const left = Math.round(screenDetailed.width  - width) / 2;
-  const top = Math.round(screenDetailed.height - height) / 2;
+  const left = screenDetailed.availLeft + Math.round(screenDetailed.availWidth - width) / 2;
```

Comment:
Where are `availLeft` and `availTop` defined? I don't see them on [ScreenDetailed](https://webscreens.github.io/window-placement/#screendetailed) or [Screen](https://drafts.csswg.org/cssom-view-1/#screen).

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Screen/availLeft
https://developer.mozilla.org/en-US/docs/Web/API/Screen/availTop

MDN reports fairly pervasive implementation (all but IE), but says they're "Non-standard: This feature is non-standard and is not on a standards track."... Perhaps we should include standardizing them on Screen (or ScreenAdvanced?) as part of the spec?

Reply:
It definitely feels wrong to reference non-standard attributes within a specification example. It seems like introducing them along with `left` and `top` in this specification is the right (or maybe the top left 😄) move.

Reply:
I added those attributes and made minor fixes (e.g. for #83). Please take another look, thanks!